mail_tracking_mailgun/models/mail_tracking_email.py

```diff
@@ -202,12 +202,18 @@ def action_manual_check_mailgun(self):
         https://documentation.mailgun.com/en/latest/api-events.html
         """
         api_key, api_url, domain, *__ = self._mailgun_values()
+        # Soft dependency to check mass mailing trackings which have their message_id
+        # in the model `mailing.trace` instead.
+        mm_tracking = hasattr(self, "mail_stats_id")
         for tracking in self:
-            if not tracking.mail_message_id:
-                raise UserError(_("There is no tracked message!"))
-            message_id = tracking.mail_message_id.message_id.replace("<", "").replace(
-                ">", ""
+            message_id = (
+                tracking.mail_message_id.message_id
+                or mm_tracking
+                and tracking.mail_stats_id.message_id
             )
+            if not message_id:
+                raise UserError(_("There is no tracked message!"))
```

Comment:
Don't put this, as there can be several reasons for not having a message_id

Reply:
Do we hide the button in such case then?

Reply:
OK

Reply:
Done